src/book/03-guides/03-hardware/02-flashing.mdx

```diff
@@ -163,3 +163,57 @@ The rest of these instructions assume a network using WPA2 (like the network use
      ```
    - Be sure to replace `<WiFi Interface>` with the name of the interface found earlier.
    - Wait a handful of seconds and run `ip addr` to check that the WiFi interface has an IP address
+
+
+## Switching Wifi Networks
+
+Switching wifi networks can be accomplished with the following two commands:
+
+```
+$ wpa_cli list_networks
+Selected interface 'wlp58s0'
+network id / ssid / bssid / flags
+0	epsilon-z	any	[CURRENT]
+1	epsilon-x	any	[DISABLED]
+2	NETGEAR89	any	[DISABLED]
+
+$ wpa_cli select_network 2
+Selected interface 'wlp58s0'
+client_loop: send disconnect: Broken pipe
+```
+
+The number `N` provided to `wpa_cli select_network <N>` is the network id given by `wpa_cli list_networks`. To switch to `epsilon-z` one would execute `wpa_cli select_network 0`.
+The example above was executed through a connection over the wifi interface, hence why the pipe broke.
+
+This approach is better than modifying the files because:
+1. Modifying files is permenent where as selecting a different network is not.
+2. Root permissions are not required to change networks.
+3. There's no need to reload the wpa_supplicant configuration.
+
+
+Finally, to check which network the robot is associated with, run:
+```bash
+$ wpa_cli status
+Selected interface 'wlp58s0'
+bssid=14:59:c0:d2:07:29
+freq=2472
+ssid=NETGEAR89
+id=2
+mode=station
+wifi_generation=4
+pairwise_cipher=CCMP
+group_cipher=CCMP
+key_mgmt=WPA2-PSK
+wpa_state=COMPLETED
+p2p_device_address=00:24:d6:d1:f6:41
+address=76:0f:5d:3d:ce:db
+uuid=f96d74c5-bf3a-5aa7-ab11-9f3892fb8d25
+```
+
+In summary, the three commands that you need to know are:
+1. `wpa_cli list_networks`
+2. `wpa_cli select_network <N>`
+3. `wpa_cli status`
+
+This technique assumes that the network has been configured in `/etc/wpa_supplicant/wpa_supplicant-wlp58s0.conf`. See above for associating with different wifi networks.
```

Comment:
```suggestion
This technique assumes that the network has been configured in `/etc/wpa_supplicant/wpa_supplicant-wlpXXsX.conf`. See above for associating with different wifi networks.
```
These numbers can change, however unlikely.

Reply:
Yeah, I guess I was trying to point out where the networks in `wpa_cli list_networks` came from. I'm not sure of a clear way of writing that though. It's sort of related to the instructions earlier in the page.